### PR TITLE
#5 Create module for members

### LINKED ISSUE #5: Create module for add and revoke members
ℹ️ This issue is part of Epic #2 

## Objective

Create a module to add and remove members of the organization

## Checklist

- [x] Add members
- [x] Revoke members
- [x] Get members

### PR DESCRIPTION
**Pull request by Jenietoc**
## Description

In this PR we created the `members` module for the management of adding, deleting and obtaining members of the organization. We also created modules necessary for the perfect functioning of the module. The `soroban_sdk` was updated to version `0.4.3`.

Fixes #5 

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] My changes generate no new warnings

### CODE REVIEW

File: mtk-automation/src/modules/member.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        let from_identifier = identifier::get_account_identifier(from.clone());
+        let member_balance = client.balance(&from_identifier);
+      
+        client.xfer_from(
+            &Signature::Invoker, 
+            &0, 
+            &from_identifier, 
+            &admin_id,
+            &member_balance
+        );
```
Comment (einerzg) [inline mtk-automation/src/modules/member.rs:48]: Please extract this code in another function!!!!

Comment (einerzg) [reply mtk-automation/src/modules/member.rs:48]: It's pending to add a swapping here!!! Please add a `todo`.

**Review by einerzg — CHANGES_REQUESTED** [review]
@Jenietoc, please check the comments and add the test!!!

**Review by L-Zuluaga — CHANGES_REQUESTED** [review]

File: mtk-automation/Cargo.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [package]
-name = "mtk-automation"
+name = "return-funds-on-leaving"
```
[the remark this answers is not in the document]
Comment (Jenietoc) [reply mtk-automation/Cargo.toml]: Already corrected it

**Review by einerzg — CHANGES_REQUESTED** [review]
@Jenietoc, please review new comments!!!

File: mtk-automation/src/contract_actions/member.rs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        client.xfer_from(
+            &Signature::Invoker, 
+            &0, 
+            &from_identifier, 
+            &admin_id,
+            &member_balance
+        );
+    }
+      
+    pub fn get_members<T: soroban_sdk::TryFromVal<Env, RawVal> + soroban_sdk::IntoVal<Env, RawVal>>(
```
[the remark this answers is not in the document]
Comment (Odraxs) [reply mtk-automation/src/contract_actions/member.rs:57]: Yes, we need it in `lib.rs` line `57`

**Review by Odraxs — APPROVED** [review]

**Review by einerzg — APPROVED** [review]
LGTM!!!

**Review by L-Zuluaga — APPROVED** [review]
LGTM! excellent job 🚀

**Pull request merged**